src/Kafka/ConsumerGroups/components/TopicsDropdown.tsx

```diff
@@ -0,0 +1,56 @@
+import { Dropdown, DropdownItem, DropdownToggle } from "@patternfly/react-core";
+import { FunctionComponent, SyntheticEvent, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+export type TopicDropdown = {
+  value: string;
+  topics: string[];
+  onChange: (topic: string) => void;
+};
+
+export const TopicDropdown: FunctionComponent<TopicDropdown> = ({
+  topics,
+  onChange,
+  value,
+}) => {
+  const { t } = useTranslation();
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const onToggle = (isOpen: boolean) => {
+    setIsOpen(isOpen);
+  };
+
+  const onSelect = (
+    event?: SyntheticEvent<HTMLDivElement, Event> | undefined
+  ) => {
+    const eventTarget = event?.target as HTMLElement;
+    const value = eventTarget.innerText;
+    onChange(value?.toLowerCase());
+    setIsOpen((prevState) => !prevState);
+  };
+
+  const dropDownItems = topics.map((topic) => (
+    <DropdownItem key={topic}>{topic}</DropdownItem>
+  ));
+
+  return (
+    <Dropdown
+      id="topic-dropdown"
+      aria-label="topic-select-dropdown"
```

Comment:
Same as above.

Reply:
My comments about using a Select component instead of Dropdown (and how to associate accessible text with the Select component) apply here as well.